### PR TITLE
[Cicognara] Use Solr8 for Production.

### PR DESCRIPTION
**Pull request by tpendragon**
Progresses https://github.com/pulibrary/cicognara-rails/issues/474

### CODE REVIEW

**Review by eliotjordan — APPROVED** [review]

File: group_vars/cicognara_production.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ rails_app_vars:
   - name: GOOGLE_CLIENT_ID
     value: '{{ vault_cicognara_google_client_id }}'
   - name: CICOGNARA_SOLR_URL
-    value: 'http://lib-solr.princeton.edu:8983/solr/cicognara'
+    value: 'http://lib-solr-prod4.princeton.edu:8983/solr/cicognara'
 sidekiq_worker_name: cicognara-workers
```
Comment (christinach) [inline group_vars/cicognara_production.yml:51]: Why not going through the load balancer `lib-solr8-prod.princeton.edu` ?

Comment (tpendragon) [reply group_vars/cicognara_production.yml:51]: Because we've had issues with that being slow in the past haven't we?

Comment (christinach) [reply group_vars/cicognara_production.yml:51]: We had with orangelight when we were pulling data from solr for the call_numbers.

Comment (christinach) [reply group_vars/cicognara_production.yml:51]: The solr_url for orangelight in the production boxes is pointing `lib-solr8-prod`

**Pull request merged**